src/SettingsPanel.tsx

```diff
@@ -72,18 +69,10 @@ const KeyboardShortcuts = () => (
         <Tr>
           <Td>
             <VStack alignItems="flex-start">
-              <Kbd>f</Kbd>
-            </VStack>
-          </Td>
-          <Td>Fit to view</Td>
-        </Tr>
-        <Tr>
-          <Td>
-            <VStack alignItems="flex-start">
-              <Kbd>r</Kbd>
+              <Kbd>Shift</Kbd> + <Kbd>1</Kbd>
```

Comment:
Remove the `VStack` entirely in favor of:

![image](https://user-images.githubusercontent.com/8332043/142257848-0fc981e9-8d3e-4ffa-96e1-47e0a0cf3ee1.png)

Reply:
Because with `VStack` the `+` is not shown in the UI